service/history/stateBuilder.go

```diff
@@ -658,7 +658,7 @@ func (b *stateBuilderImpl) scheduleWorkflowTimerTask(event *shared.HistoryEvent,
 	timeout := now.Add(time.Duration(msBuilder.GetExecutionInfo().WorkflowTimeout) * time.Second)
 
 	cronSchedule := b.msBuilder.GetExecutionInfo().CronSchedule
-	cronBackoffDuration := backoff.GetBackoffForNextSchedule(cronSchedule, now)
+	cronBackoffDuration := backoff.GetBackoffForNextSchedule(cronSchedule, now, now)
```

Comment:
Let's talk offline about this. I am not sure this is correct.

Reply:
Changed to get this from start event